test/functional/tests/cli/test_cli_standby.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright(c) 2019-2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas.cli import casadm_bin
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from test_utils.size import Size, Unit
+from api.cas.cli_messages import check_stderr_msg, missing_param, disallowed_param
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+def test_activate_neg_cli_params():
+    """
+    title: Verifying parameters for activating a standby cache instance.
+    description: |
+        Try executing the standby activate command with required arguments missing or unallowed
+        arguments present.
+    pass_criteria:
+        -The execution is unsuccessful for all improper argument combinations
+        -A proper error message is displayed for unsuccessful executions
+    """
+    with TestRun.step("Prepare the device for the cache."):
+        cache_device = TestRun.disks["cache"]
+        cache_device.create_partitions([Size(500, Unit.MebiByte)])
+        cache_device = cache_device.partitions[0]
+        cache_id = 1
+        cache_line_size = 32
+
+    with TestRun.step("Init standby cache"):
+        cmd_init = casadm_bin + f"--standby --init --cache-id {cache_id} " \
+                                f"--cache-device {cache_device.path} --cache-line-size" \
+                                f" {cache_line_size} --force"
+        output = TestRun.executor.run(cmd_init)
+        if output.exit_code == 0:
+            TestRun.LOGGER.error(f'"Standby activate failed with command {cmd_init}"')
```

Comment:
Consider using `standby_init()` function from `api/cas/casadm.py`. It wraps setting up a command and handling error. Additionally it returns `Cache` object.